### ISSUE TITLE
Makefile (parallel make?) instability

### ISSUE THREAD

**Issue by bsdphk** [varnishcache/varnish-cache#3370]
VTEST shows instability, possily from parallel make.

**Comment by nigoroll** [comment]
Taking it because it smells like related to #3310

**Comment by nigoroll** [comment]
FTR: http://varnish-cache.org/vtest/attachment_6ba917d81d4df3ab9f4d4483c5a62a7851184bba_FreeBSD_amd64_12.1-REL_1.04__phk_distcheck.txt

```
--- link_srcdir ---
if test "x../../../../doc/sphinx" != "x." && test ! -f index.rst; then  s=`realpath ../../../../doc/sphinx`;  for f in `cd $s && find . -type f`; do  d=`dirname $f`;  test -d $d || mkdir -p $d;  test -f $f || ln -s $s/$f $f;  done  fi
--- include/varnishstat_bindings.rst ---
mv include/varnishstat_bindings.rst_ include/varnishstat_bindings.rst
--- include/cli.rst ---
../../bin/varnishd/varnishd -x cli > include/cli.rst_
mv include/cli.rst_ include/cli.rst
--- link_srcdir ---
ln: ./include/cli.rst: File exists
--- include/counters.rst ---
--- link_srcdir ---
*** [link_srcdir] Error code 1
```

**Comment by nigoroll** [comment]
my attempt to fix this has failed, I gave up and reverted all changes.
see 3f2be2ce824e7e13fcc930b6185fe414e701f40e

**Comment by Dridi** [comment]
This is most likely solved by #3443.

See https://github.com/varnishcache/varnish-cache/commit/4c872289ee88e4460c3caff00154489fd4e64a82 that has since been squashed after @bsdphk's test of #3443 on FreeBD.

**Comment by Dridi** [comment]
It should be fixed by #3443, please reopen if it still happens.
